### ISSUE TITLE
Fetch success result "Title" not spoken in response

### ISSUE THREAD

**Issue by j4cknife** [d8ahazard/Phlex#81]
I'm able to add shows and movies to sonarr and radarr to download but when Flex TV responds with success status "I've added X added to the fetch list." it's always null (). Example from the Phlex.log below.

```Sun, 11 Jun 2017 02:22:33 +0200 [ DEBUG ] api.php::parseFetchCommand - Final result: {"status":"SUCCESS: Media added to searcher.","mediaResult":{"@attributes":{"title":"The Neon Demon","sortTitle":"neon demon","sizeOnDisk":0,"status":"tba","images":[{"coverType":"poster","url":"\/MediaCover\/42\/poster.jpg?lastWrite=636327373530000000"}],"downloaded":false,"year":2016,"hasFile":false,"path":"z","profileId":3,"pathState":"static","monitored":true,"minimumAvailability":"tba","isAvailable":true,"folderName":"The Neon Demon.(2016)","runtime":0,"cleanTitle":"theneondemon","tmdbId":301365,"titleSlug":"the-neon-demon-301365","genres":[],"tags":[],"added":"2017-06-11T00:22:33.461275Z","addOptions":{"searchForMovie":true,"ignoreEpisodesWithFiles":false,"ignoreEpisodesWithoutFiles":false},"alternativeTitles":[],"qualityProfileId":3,"id":42,"art":"http:\/\/localhost:7878\/api\/MediaCover\/42\/banner.jpg?apikey=z","type":"movie"}},"mediaStatus":"SUCCESS: Media added to searcher.","parsedCommand":"Fetch the first movie or show named neon demon","initialCommand":"neon demon"}
Sun, 11 Jun 2017 02:22:33 +0200 [ DEBUG ] api.php::parseApiCommand - MediaResult: {"status":"SUCCESS: Media added to searcher.","mediaResult":{"@attributes":{"title":"The Neon Demon","sortTitle":"neon demon","sizeOnDisk":0,"status":"tba","images":[{"coverType":"poster","url":"\/MediaCover\/42\/poster.jpg?lastWrite=636327373530000000"}],"downloaded":false,"year":2016,"hasFile":false,"path":"z","profileId":3,"pathState":"static","monitored":true,"minimumAvailability":"tba","isAvailable":true,"folderName":"The Neon Demon.(2016)","runtime":0,"cleanTitle":"theneondemon","tmdbId":301365,"titleSlug":"the-neon-demon-301365","genres":[],"tags":[],"added":"2017-06-11T00:22:33.461275Z","addOptions":{"searchForMovie":true,"ignoreEpisodesWithFiles":false,"ignoreEpisodesWithoutFiles":false},"alternativeTitles":[],"qualityProfileId":3,"id":42,"art":"http:\/\/localhost:7878\/api\/MediaCover\/42\/banner.jpg?apikey=z","type":"movie"}},"mediaStatus":"SUCCESS: Media added to searcher.","parsedCommand":"Fetch the first movie or show named neon demon","initialCommand":"neon demon"}
Sun, 11 Jun 2017 02:22:33 +0200 [ DEBUG ] api.php::parseApiCommand - Should have been successful.
Sun, 11 Jun 2017 02:22:33 +0200 [ DEBUG ] api.php::parseApiCommand - Success and not in searcher?
Sun, 11 Jun 2017 02:22:33 +0200 [ DEBUG ] api.php::returnSpeech - Final Speech should be: Okay, I've added  () to the fetch list.```

**Comment by j4cknife** [comment]
On what OS are you running Phlex?
**ubuntu 16.04.2 LTS**

Are you using a new instance of XAMPP, or an existing webserver?
**new xampp**

If Using an existing Webserver, have you ensured the proper modules are enabled and installed, and that PHP can read and write to the /Phlex directory?
**777 set for /Phlex**

If you're having issues communicating with a specific instance of Plex/Couchpotato/Sonarr/etc. - tell me the OS you're running that app on.
**sonarr and radarr are local to the same server Phlex is running ubuntu server**

Is Phlex using http or https?
**Phlex is using http**

**Comment by d8ahazard** [comment]
Fixed with #3e039b4
